service/src/main/frontend/README.md

```diff
@@ -0,0 +1,87 @@
+# <p style="margin-bottom: 0px">Datawave Data Dictionary Version 2 <img src="./public/favicon.ico" style="width: 2em; height: 2em; vertical-align: middle;"></p>
+
+### Project Overview
+Data Dictionary Version 2 is a new microservice designed to enhance the original Data Dictionary. It leverages an updated tech stack to incorporate features that facilitate data analysis and provide quick query functions. Below are detailed instructions on how to install, run, and configure this project. This README will guide you through the process of starting and running the project.
+
+### Tech Stack Overview
+The tech stack for this project includes **Vue.js (v3.0)** with **Vite** for building the frontend and enabling hot module replacement. **Quasar (v2.8)** is used for constructing the main table and handling data insertion. The backend is powered by **Java 11** with **Spring Boot** for serving data through RESTful methods, while **Node.js (v16+)** and **NPM** manage packages and dependencies. Lastly, **Apache Accumulo** provides data storage for visualization.
+
+## Running the Application
+This section provides the installation details and outlines the steps to run Data Dictionary V2 in production-ready mode. Additionally, it includes instructions for making rapid frontend changes using Quasar's Dev Mode with Vue. While both modes can be operated independently, it is strongly recommended to conduct thorough testing in production mode, as it closely simulates the deployment environment.
+
+### Instructions for Production-Ready Container Mode
+1. To run the application in production mode, first ensure Java 11 and Maven is installed. Next, clone the [Datawave Repository](https://github.com/NationalSecurityAgency/datawave) into your preferred directory, then navigate into it by running the following command(s):
+
+    ```
+    git clone git@github.com:NationalSecurityAgency/datawave.git
+    cd /path/to/your/directory/datawave
```

Comment:
Minor nitpick, you should be able to assume datawave was cloned into a subdirectory of the current directory.
```suggestion
    cd datawave
```